code/modules/clothing/gloves/color.dm

```diff
@@ -12,6 +12,41 @@
 	custom_price = 1200
 	custom_premium_price = 1200
 
+/obj/item/toy/sprayoncan
+	name = "spray-on insuls applicator"
+	desc = "What is the number one problem facing our station today?"
+	icon = 'icons/obj/clothing/gloves.dmi'
+	icon_state = "sprayoncan"
+
+/obj/item/toy/sprayoncan/afterattack(atom/target, mob/living/carbon/user, proximity)
+	if(iscarbon(target) && proximity)
+		var/mob/living/carbon/C = target
+		var/mob/living/carbon/U = user
+		if(C == user)
+			C.visible_message("<span class='notice'>[U] sprays their hands with glittery rubber!</span>")
+		else
+			C.visible_message("<span class='warning'>[U] sprays glittery rubber on the hands of [C]!</span>")
+		C.equip_to_slot_or_del(new /obj/item/clothing/gloves/color/yellow/sprayon,ITEM_SLOT_GLOVES) //we're gonna pretend that the rubber can't stick to gloves.
+		qdel(src)
+
+/obj/item/clothing/gloves/color/yellow/sprayon
+	desc = "How're you gonna get 'em off, nerd?"
+	name = "spray-on insulated gloves"
+	icon_state = "sprayon"
+	item_state = "sprayon"
+	permeability_coefficient = 0
+	resistance_flags = ACID_PROOF
+	var/shocks_remaining = 10
+
+/obj/item/clothing/gloves/color/yellow/sprayon/Initialize()
+	.=..()
+	ADD_TRAIT(src, TRAIT_NODROP, CURSED_ITEM_TRAIT) //Cursed with the power of science, or something

```

Comment:
I dislike this being an actual item that blocks other gloves, would make more sense as a trait or a component that grants a trait

Reply:
but that'd destroy the reference pkpenguin

Reply:
wait so you could cut off your arm and get an actual pair of gloves that someone else could but on?

Reply:
For PKPenguin, it's intended behavior to make it block other gloves. They're supposed to be unremovable and unreplaceable as according to their reference. If you really wanted to, it would be pretty simple to just shock yourself 10 times for them to break.
And for itseasytosee, I'm just gonna chalk it up to SS13 logic, since you also need two arms to wear a gloves at all.

Reply:
I guess it works for the reference. They should delete when dropped though since generally you don't want NODROP items lying around

Reply:
i see where you're coming from, fixed